modules/nw-rosa-proxy-remove-cli.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * networking/enable-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rosa-proxy-remove-cli_{context}"]
+= Removing the cluster-wide proxy using CLI
+
+You must use the `rosa` CLI to remove the proxy's address from your cluster.
+
+.Prerequisites
+
+* Cluster administrator permissions
+* {product-title} `rosa` CLI tool installed
+
+.Procedure
+
+. Use the `rosa edit` command to modify the proxy. You must pass empty strings to the `--http-proxy` and `--https-proxy` arguments to clear the proxy from the cluster:
++
+[source,terminal]
+----
+$ rosa edit cluster -c <cluster_name> --http-proxy "" --https-proxy "" 
+----
++
+[NOTE]
+====
+While your proxy may only use one of the proxy arguments, the empty field is ignored, so passing empty strings to both the `--http-proxy` and `--https-proxy` does not cause any issues.
+====
+
+
+. You receive the following response:
++
+.Sample Output
++
+[source,yaml]
+----
+I: Updated cluster <cluster_name>
+----
+
+.Verification
+
+. You can verify that the cluster has been removed with the `rosa describe` command. 
++
+[source,yaml]
+----
+rosa describe cluster -c <cluster_name>
+----
+
+. Before removal, the proxy IP displays in a proxy section:
```

Comment:
This step and the next one also seem more like example output than actual steps. If so, consider just attaching them to the previous step with `+`s as additional paragraphs. And same comment as previous re: converting to a single-step procedure (from `.` to `*`).